src/app/components/navbar/index.tsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+
+import React from "react";
+import Link from "next/link";
+import Wrapper from "../wrapper";
+import { CustomButton } from "../custom-button/CustomButtom";
+import { Cross1Icon, ExternalLinkIcon, HamburgerMenuIcon } from "@radix-ui/react-icons";
+
+export const Navbar = () => {
+  const [openMenu, setOpenMenu] = React.useState(false);
+
+  return (
+    <div className='w-full flex items-center justify-center sticky top-0 bg-secondary-black z-50'>
+      <Wrapper className='w-full'>
+        <div className='h-[68px] md:h-[100px] w-full flex items-center justify-between'>
+          <h3 className=' text-xl sm:text-2xl font-extrabold leading-9 tracking-[0.96px]'>
+            MAVAPAY.<span className='text-primary-green'>MONEY</span>
+          </h3>
+          <section className='hidden md:flex items-center gap-5 md:gap-10 text-sm md:text-base'>
+            <Link href='' className='tracking-[0.32px]'>
```

Comment:
lets link this to the contact us section
```suggestion
            <Link href='#contact-us' className='tracking-[0.32px]'>
```